tests/functional/spec/features/hub_and_spoke/hub_and_spoke.spec.js

```diff
@@ -40,6 +40,12 @@ describe("Feature: Hub and Spoke", () => {
       await expect(await $(HubPage.summaryRowState("household-section")).getText()).toBe("Not started");
     });
 
+    it("When a user has a screen reader, The visually hidden text should read the name and state of each section in the hub", async () => {
```

Comment:
I would consider updating this as to me it implies there is some specific logic in place for screen readers, but this is the case for all users.

Reply:
Rephrased to "When a user utilises a screen reader, The visually hidden text read aloud should be the state and name of each section in the hub"